pkg/controller/podgroup.go

```diff
@@ -238,6 +238,9 @@ func (ctrl *PodGroupController) syncHandler(key string) error {
 					failed++
 				}
 			}
+		} else {
+			pgCopy.Status.Phase = schedv1alpha1.PodGroupPending
+			break
 		}
 		pgCopy.Status.Failed = failed
```

Comment:
If the number of pods of the group is scaled to 0, we also need to update this information.
```go
pgCopy.Status.Failed = failed
pgCopy.Status.Succeeded = succeeded
pgCopy.Status.Running = running
```

Reply:
done